dlab_core/infrastructure/repositories.py

```diff
@@ -206,29 +332,51 @@ def _silence_stderr():
             sys.stderr = old_target
 
     def _load_data(self):
+        """Load data from data source.
+
+        :rtype: list of dict
+        :return: Repository data.
+
+        :raises: RepositoryException
+        """
+
         try:
             with self._silence_stderr():
                 args = self._arg_parse.parse_args()
         except SystemExit:
-            raise DLabException(self.LC_ERR_WRONG_ARGUMENTS)
+            raise RepositoryException(self.LC_ERR_WRONG_ARGUMENTS)
 
         for key, val in vars(args).items():
             self._data[key] = val
 
     def add_argument(self, *args, **kwargs):
+        """Add argument parser argument."""
+
         self._arg_parse.add_argument(*args, **kwargs)
         self._data = {}
 
 
 class ConfigRepository(BaseFileRepository, BaseLazyLoadRepository):
+    """Repository based on file data.
+
+    :type location: str
+    :param location: Data source file location.
+    """
+
     VARIABLE_TEMPLATE = "{0}_{1}"
 
-    def __init__(self, absolute_path):
-        super(ConfigRepository, self).__init__(absolute_path)
+    def __init__(self, location):
+        super(ConfigRepository, self).__init__(location)
 
     def _load_data(self):
+        """Load data from data source.
+
+        :rtype: list of dict
+        :return: Repository data.
+        """
+
         config = ConfigParser()
-        config.read(self.file_path)
+        config.read(self.location)
         for section in config.sections():
             for option in config.options(section):
                 var = self.VARIABLE_TEMPLATE.format(section, option)
```

Comment:
line 383
if var not in self._data.keys(): -> if var not in self._data
google style guide says it's better https://github.com/google/styleguide/blob/gh-pages/pyguide.md#284-decision